Support schildichat protocol

AndrewRyanChama opened the pull request:
Currently, the desktop app supports element:// but there is no specific
protocol for schildichat.  This change adds support for schildichat:// in
addition to element://

Tested on crapos but not linux or windows
schildichat://vector/webapp/#/room%2F%23web%3Aschildi.chat

<!-- CHANGELOG_PREVIEW_START -->
---
This PR currently has no changelog labels, so will not be included in changelogs.

A reviewer can add one of: `T-Deprecation`, `T-Enhancement`, `T-Defect`, `T-Task` to indicate what type of change this is, or add `Type: [enhancement/defect/task]` to the description and I'll add them for you.<!-- CHANGELOG_PREVIEW_END -->

su-ex commented on package.json:
```diff
@@ -147,6 +147,12 @@
         ]
+      },
+      {
+        "name": "schildichat",
```
This should go into `overlay-package.json`. We apply this on top of the normal `package.json` to ease merges: https://github.com/SchildiChat/schildichat-desktop/blob/sc/merge_helpers.sh#L218-L228

AndrewRyanChama committed:
Support schildichat protocol

Currently, the desktop app supports element:// but there is no specific
protocol for schildichat.  This change adds support for schildichat:// in
addition to element://

su-ex reviewed (approved):
Looks good. :+1: 
I'm currently in the process of merging upstream, thus I'll merge this later.
